app/controllers/runs_controller.rb

```diff
@@ -79,7 +83,7 @@ def get_runner
   end
 
   def run_params
-    params.require(:run).permit(:runner_id, :date, :training_plan, :progress)
+    params.require(:run).permit(:runner_id, :date, :training_plan, :progress, :changed_flag)
```

Comment:
Changed to customize_flag: https://github.com/o19s/thats-trackable/commit/66e7a40f628a29d42fd91669b0f6c09ea13ef9ab AND https://github.com/o19s/thats-trackable/commit/a7a0445e5cbcd18357bcade07c6c13f98c075d26